addon/ng2/commands/test.ts

```diff
@@ -14,7 +14,7 @@ module.exports = TestCommand.extend({
   ],
 
   run: function (commandOptions) {
-    this.project.ngConfig = this.project.ngConfig || config.CliConfig.fromProject();
+    this.project.ngConfig = this.project.ngConfig || CliConfig.fromProject();
```

Comment:
But won't it break other commands if they are run in sequence? Everything else expects `this.project.ngConfig` to have `config` instead. 

Or the other way around, if nothing else breaks by having one command assign the wrong thing to `this.project.ngConfig`, maybe this is unnecessary.